android/src/ti/map/AnnotationProxy.java

```diff
@@ -340,33 +356,22 @@ public void onPropertyChanged(String name, Object value)
 			updateInfoWindow();
 		} else if (name.equals(TiC.PROPERTY_LEFT_BUTTON)) {
 			getOrCreateMapInfoWindow().setLeftOrRightPane(value, TiMapInfoWindow.LEFT_PANE);
-			if (value == null) {
-				Object leftView = getProperty(TiC.PROPERTY_LEFT_VIEW);
-				if (leftView != null) {
-					getOrCreateMapInfoWindow().setLeftOrRightPane(leftView, TiMapInfoWindow.LEFT_PANE);
-				}
-			}
```

Comment:
Hey Hans. I've removed it because it has it's own conditional on line 360. It was duplicated.